mmdet3d/models/backbones/pointnet2_sa_msg.py

```diff
@@ -103,7 +106,7 @@ def __init__(self,
                     sa_out_channel,
                     aggregation_channels[sa_index],
                     conv_cfg=dict(type='Conv1d'),
-                    norm_cfg=dict(type='BN1d'),
+                    norm_cfg=dict(type='BN1d', eps=1e-3),
```

Comment:
Put the eps argument in the config rather than in the default arguments. Also, if it is not necessary, we do not need use them.